master/internal/command/command.go

```diff
@@ -352,13 +352,37 @@ func (c *command) readinessChecksPass(ctx *actor.Context, log sproto.ContainerLo
 	return len(c.readinessChecks) == 0
 }
 
+func (c *command) getState() State {
```

Comment:
I'd just make this `State()` and doc it. Meant to leave this the first time around oops.